src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+form {
+  position: relative;
+  margin: 20px 7px;
```

```suggestion
  margin: 20px auto;
```
You need only vertical margins. Horizontal is not important